### PR TITLE
[converter] fix quantization issue of post-fusion strategy of conv2d-gather sequences & test-cases

### PR DESCRIPTION
**Pull request by xindongzhang**
Fix the quantization issue of conv2d-gather, as discussed in DingTalk
dingtak

### CODE REVIEW

File: tests/converter_op_test.py [hunk 1]
```diff
@@ -3455,6 +3455,38 @@ def forward(self, x):
         tfl_output = tfl_run_model(model_path, dummy_input, dummy_output)
         assert_close(dummy_output, tfl_output)
 
+    @unittest.skipIf(not hasattr(torch.nn.quantized, 'Conv2d'), 'Quantized conv2d is not supported')
```
Comment (peterjc123) [inline tests/converter_op_test.py:3458]: It seems that you forgot to remove this function, right?

**Review by peterjc123 — CHANGES_REQUESTED** [review]

File: tinynn/converter/operators/optimize.py [hunk 2]
```diff
@@ -352,23 +352,37 @@ def fuse_conv2d_gather(self):
...
             actions.append((self.graph.replace_operator_input, (conv, 1, new_w)))
 
             # permute bias of conv-op
             b = conv['op'].inputs[2].tensor.copy() if len(conv['op'].inputs) > 2 else None
+            b_quant_param = conv['op'].inputs[2].quantization
             new_b = None
             if b is not None:
                 new_b = np.take(b, indx, axis=0)
-                new_b = self.create_attr_tensor(new_b)
+                if b_quant_param is not None and isinstance(b_quant_param.scale, list) and b_quant_param.dim == 0:
```
Comment (peterjc123) [inline tinynn/converter/operators/optimize.py:380]: `b_quant_param.dim == 0` is not correct. Since it is always true (`b` has only one dimension), if `w_quant_param.dim !=0` then you are swapping `b` but not `w`, this is not right because they should be swapped together. 

Apparently, this correct logic is that when the condition in line 366 is satisfied, then we perform the logic in line 381-385.

File: tests/converter_op_test.py (hunk 1 above)
Comment (xindongzhang) [reply tests/converter_op_test.py]: Thanks for your kindly reminder, pls kindly check the revision.

File: tinynn/converter/operators/optimize.py (hunk 2 above)
Comment (xindongzhang) [reply tinynn/converter/operators/optimize.py]: Thanks for the detailed comment.

**Review by peterjc123 — APPROVED** [review]
